test/var_compatibility_test/rad_sw.F90

```diff
@@ -26,8 +26,8 @@ subroutine rad_sw_run(ncol, fluxSW, errmsg, errflg)
     errflg = 0
 
     do icol=1,ncol
-       fluxSW(icol)%sfc_up_sw   = 100._kind_phys
-       fluxSW(icol)%sfc_down_sw = 400._kind_phys
+       sfc_up_sw(icol)   = 100._kind_phys
+       sfc_down_sw(icol) = 400._kind_phys
```

Comment:
Are we checking somewhere that these are actually being set correctly? If not, we should check that! To ensure the framework is not just creating new interstitial variables and not grabbing them from the appropriate DDT

Reply:
Done.